feat: UIG-2769 - form components storybook docs

ksvlaanderen opened the pull request:
[bamboo](https://www.milieuinfo.be/bamboo/browse/UIGOV-CUWC192)
[jira](https://www.milieuinfo.be/jira/browse/UIG-2769)

Goldflow reviewed (dismissed):
👍🏼

Goldflow reviewed (commented):
code veranderingen zelf zien er goed uit maar er is nog een issue met build op bamboo

ksvlaanderen committed:
feat: UIG-2769 - form components storybook docs

Form components storybook docs aangepast
Cypress reporter verwijderd uit e2e apps
cypress-multi-reporters verwijderd uit package.json

Goldflow reviewed (approved):
goed!
